pkgs/misc/drivers/xow/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, libusb, gitMinimal }:
+
+stdenv.mkDerivation rec {
+  pname = "xow";
+  version = "0.2";
+
+  src = fetchFromGitHub {
+    owner = "medusalix";
+    repo = "xow";
+    rev = "v${version}";
+    sha256 = "03ajal91xi52svzy621aa4jcdf0vj4pqd52kljam0wryrlmcpbr3";
+  };
+
+  makeFlags = "BUILD=RELEASE";
+  enableParallelBuilding = true;
+  nativeBuildInputs = [ gitMinimal ];
+  buildInputs = [ libusb ];
+
+  installPhase = ''
```

Comment:
When overriding `installPhase`, you should also provide `runHook preInstall` and `runHook postInstall`. Though, in this case, it is better to rely on the default `installPhase`.

Reply:
Since the author surely won't fix this in 0.2, I have added the hooks for now.